notebooks/attack_parallel_auto_attack.ipynb

```diff
@@ -0,0 +1,2146 @@
+{
```

Comment:
Could you please add a descriptive title and add a few comments between the most important cells?